File: Lab2_Qn 1a.py
#avg distance in an array

i=0
sum=0
t=1
sum1=0

#Calculating front distance with unit of distance=1 unit
while i<=5:
    t=1
    while t<=(6-i):
        sum=sum+(t-1)
        print(sum)
        t=t+1
    sum1=sum
    print('heyyy' + str(sum1))
    i=i+1

#Back distance is same as front distance

y=2 * sum
print('final ' + str(y))
avg=y/(6*6)
print(avg)

File: Lab2_Qn1a.cpp
//To find avg distance in an array

#include<iostream>
using namespace std;
int main(){

int arr[]={1,2,3,4,5,6};

int i=1;
int j=1;
float d=0;
while (i<6)
    {
        j=i;
        while (j<=6)
        {
            
            d=d+ abs(i-j)+ abs(j-i);
            //cout<<"***"<<d<<endl;
            j=j+1;
        }
        i=i+1;
    }
cout<<d<<endl;
float final=d/36;
cout<<final<<endl;
return 0;
}


